dll/cpl/sysdm/virtmem.c

```diff
@@ -442,8 +445,9 @@ OnSet(PVIRTMEM pVirtMem)
             }
 
             /* Check the valid range of the minimum size */
-            if (MinSize < 16 ||
-                MinSize > pVirtMem->Pagefile[DriveIndex].FreeSize)
+            if (MinSize < 2 ||
+                MinSize > pVirtMem->Pagefile[DriveIndex].FreeSize ||
+                MinSize > 4096)
```

Comment:
Please define a macro (such as `MINSIZE_MAX`) for this magic number.

Reply:
No benefit (used once)